README.md

```diff
@@ -1 +1,16 @@
-# hw01
\ No newline at end of file
+
+### Brief Biography Arline  
+My name is Arline Lee. I am a museum science teacher. I am unabashedly and unironically a huge fan of: 
+* **Science Fiction**  
+   + I have recently come around to some genre permeability with fantasy
+      + If anyone is interested N.K. Jemisin's *The Fifth Seasson* is a perfect example of "hard sci-fi" fantasy. 
+  + My favorite series is *Deep Space Nine*, but Captain Janeway is my favorite captain 
```

Comment:
Agree with DS9. It has that serialized approach and darker themes than prior trek, without going completely off the rails like the newest series. I'm still halfway through Picard - after seeing Icheb's eye ripped out, I'm not particularly interested in finishing it. But I really want to see Riker again